src/api/forms/service.js

```diff
@@ -289,10 +288,14 @@ export async function updateFormMetadata(formId, formUpdate, author) {
     await formMetadata.update(formId, { $set: updatedForm })
     logger.info(`Updated form metadata for form ID ${formId}`)
 
-    return updatedForm.slug ?? form.slug
+    return { slug: updatedForm.slug ?? form.slug }
   } catch (err) {
+    if (err.name === 'MongoServerError' && err.code === 11000) {
+      return {
+        error: new Error(`Form title ${formUpdate.title} already exists`)
```

Comment:
You're already using Boom on like 271, why don't we use the same mechanism for throwing errors rather than building something new?